test/LaunchDarkly.EventSource.Tests/Stubs/EventSink.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Concurrent;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace LaunchDarkly.EventSource.Tests
+{
+    public class EventSink
```

Comment:
We can use this class and some of the other approaches I used in the newer tests to streamline the existing tests quite a bit, but I'll save that for another PR.